vignettes/ees-api-workflow.Rmd

```diff
@@ -174,23 +174,28 @@ eesyapi::get_meta(dataset_id = "7c0e9201-c7c0-ff73-bee4-304e731ec0e6") |>
 
 ## Retrieving data from a data set
 
-### Downloading an entire data set in a single file
+### Preview raw CSV data
 
-Whilst specific queries (as detailed in the following sections) are best used for automated pipeline 
-workflows, the EES API also offers the option
-to just download the entirety of the underlying data file for a given data set. This bypasses the 
-use of IDs, making it a simpler short-term option to retrieve data, but with the risk of creating
-breaking changes in automated pipelines should element labels change.
+In the EES API you have the option to preview underlying data CSV file for a given data set. 
```

Comment:
"preview underlying data" -> "preview the underlying data".
Also wondering whether the end user needs to know / care that it's a csv file behind this? Might just be extra info that they don't need to know here.

Reply:
Makes sense, updated in PR response commit